aspnetcore/blazor/performance/app-download-size.md

```diff
@@ -89,7 +95,7 @@ A data file is included to make timezone information correct. If the app doesn't
 
 ```xml
 <PropertyGroup>
-<BlazorEnableTimeZoneSupport>false</BlazorEnableTimeZoneSupport>
+  <BlazorEnableTimeZoneSupport>false</BlazorEnableTimeZoneSupport>
```

Comment:
we have `InvariantTimezone` and `BlazorEnableTimeZoneSupport`

@maraf do we have preference. Do both work ?

Reply:
I found out that we're supposed to only be describing `<InvariantTimezone>`, per guidance in our Glob-Loc article ...

> > [!NOTE]
> `<BlazorEnableTimeZoneSupport>` overrides an earlier `<InvariantTimezone>` setting. We recommend removing the `<BlazorEnableTimeZoneSupport>` setting.